diff -Naur fishtest-master/fishtest/fishtest/__init__.py fishtest/fishtest/fishtest/__init__.py
--- fishtest-master/fishtest/fishtest/__init__.py 2016-01-03 18:42:35.000000000 +0100
+++ fishtest/fishtest/fishtest/__init__.py    2016-01-06 22:12:12.404453300 +0100
@@ -66,4 +66,14 @@
   config.add_route('api_request_spsa', '/api/request_spsa')

   config.scan()
+  
+  # IMPORTANT: use this code only to initialize a development site
+  if not rundb.userdb.get_user('user00'):
+    # add user00 as approver
+    rundb.userdb.create_user('user00', 'user00', 'user00@user00.user00')     
+    rundb.userdb.add_user_group('user00', 'group:approvers')
+    #add user01 as normal user
+    rundb.userdb.create_user('user01', 'user01','user01@user01.user01')
+  ###
+  
   return config.make_wsgi_app()
